Lib/fontTools/ttLib/tables/_g_l_y_f.py

```diff
@@ -841,7 +841,10 @@ def getCoordinates(self, glyfTable):
 			allEndPts = []
 			for compo in self.components:
 				g = glyfTable[compo.glyphName]
-				coordinates, endPts, flags = g.getCoordinates(glyfTable)
+				try:
+					coordinates, endPts, flags = g.getCoordinates(glyfTable)
+				except RuntimeError:
```

Comment:
python3.5 has `RecursionError` built-in exception, maybe add a TODO for later as we drop py27 support.
should you do instead:

```python
except RuntimeError as e:
    if e.args[0] == "maximum recursion depth exceeded":
        raise ttLib.TTLibError(...)
    raise
```

Reply:
Or maybe: add RecursionError as an alias for RuntimeError in py23?

Reply:
No because RuntimeError is broader than RecursionError

Reply:
I understand, but: the alias would only be for 2.7, and in Python 3 we'd get the "correct" exception. That means, yes, we catch a little broader on 2.7, but there'll be no todo for Python 3, and no deed to parse the exception message. For me that's 2 pluses for Py3 and one slight minus for Py2.

Reply:
Fair enough. It’ll be a temporary hack anyway.